### PR TITLE
fix: 2560 include workflow updates in release branch

### PR DESCRIPTION
**Pull request by ebadiere**
Cherry pick release automation fixes into the release branch

**Related issue(s)**: #2560 

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by rbarkerSL — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 Implementation of an Ethereum JSON RPC APIs for Hedera Hashgraph. Utilises both Hedera Consensus Nodes and Mirror nodes
 to support RPC queries as defined in
-the [JSON RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=false&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false)
+the [JSON RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/release/0.49/docs/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=false&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false)
```
Comment (Nana-EC) [inline README.md:19]:
```suggestion
the [JSON RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/hashgraph/hedera-json-rpc-relay/main/docs/openrpc.json&uiSchema%5BappBar%5D%5Bui:splitView%5D=false&uiSchema%5BappBar%5D%5Bui:input%5D=false&uiSchema%5BappBar%5D%5Bui:examplesDropdown%5D=false)
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Wrong branch, this should be pointing to release/0.49

File: README.md (hunk 1 above)
Comment (ebadiere) [reply README.md]: Updated

**Review by Nana-EC — APPROVED** [review]
LG

**Pull request merged**